cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/UserManagementTest.scala

```diff
@@ -67,6 +67,14 @@ ALTER USER alice SET STATUS SUSPENDED
 
 (★) Change the user status to suspended. Use `SET STATUS ACTIVE` to reactivate the user.
 
+###dontrun
+//
+
+ALTER USER alice SET HOME DATABASE otherDb
+###
+
+(★) Change the home database of user to otherDb. Use `REMOVE HOME DATABASE` to have the default database be home for the user again.
```

Comment:
`... to have the default database be home for ...` sounds odd

Reply:
yes and I have no idea on a better wording...

Reply:
```suggestion
(★) Change the home database of user to otherDb. Use `REMOVE HOME DATABASE` to unset the home database for the user and fallback to the default database.
```
```suggestion
(★) Change the home database of user to otherDb. Use `REMOVE HOME DATABASE` to unset the home database for the user and have the default database again.
```
```suggestion
(★) Change the home database of user to otherDb. Use `REMOVE HOME DATABASE` to unset the home database for the user and use the default database again.
```
```suggestion
(★) Change the home database of user to otherDb. Use `REMOVE HOME DATABASE` to unset the home database for the user and rely on the default database again.
```
I don't know 🤷